src/Compilers/CSharp/Portable/PublicAPI.Unshipped.txt

```diff
@@ -211,6 +297,9 @@ static Microsoft.CodeAnalysis.CSharp.SyntaxFactory.WithExpression(Microsoft.Code
 *REMOVED*Microsoft.CodeAnalysis.CSharp.Syntax.ObjectCreationExpressionSyntax.ArgumentList.get -> Microsoft.CodeAnalysis.CSharp.Syntax.ArgumentListSyntax
 *REMOVED*Microsoft.CodeAnalysis.CSharp.Syntax.ObjectCreationExpressionSyntax.Initializer.get -> Microsoft.CodeAnalysis.CSharp.Syntax.InitializerExpressionSyntax
 *REMOVED*Microsoft.CodeAnalysis.CSharp.Syntax.ObjectCreationExpressionSyntax.NewKeyword.get -> Microsoft.CodeAnalysis.SyntaxToken
+*REMOVED*Microsoft.CodeAnalysis.CSharp.Syntax.ParameterSyntax.AttributeLists.get -> Microsoft.CodeAnalysis.SyntaxList<Microsoft.CodeAnalysis.CSharp.Syntax.AttributeListSyntax>
```

Comment:
Doing this and _also_ updating the PublicAPI.Shipped.txt file? I'm a bit confused.

Reply:
I'm not sure what the quickfix did here, I'll double check.